src/RemoteStoragesList/RemoteStoragesListContainer.js

```diff
@@ -36,18 +36,15 @@ const RemoteStoragesListContainer = ({
   const loadConfigurations = useCallback(() => {
     setIsLoading(true);
     mutator.configurations.GET().then(({ totalRecords, configurations }) => {
-      setStoragesList(prev => [
-        ...prev,
```

Comment:
you don't need to support portions now?

Reply:
I guess it hasn't been supported here anyway. No "load next" or anything similar.
In fact it's not needed here, for the configurations: not so many of them are supposed to exist.